R/utils.R

```diff
@@ -1,11 +1,20 @@
 #'@importFrom httr stop_for_status GET user_agent content status_code
 pv_query <- function(params, reformat = TRUE, ...){
+  if(length(params) > 1){
+    data <- lapply(as.list(params), pv_query, reformat = reformat)
+    if(reformat == TRUE){
+      return(do.call(rbind, data))
+    } else {
+      return(data)
```

Comment:
Sure, that's what I meant; TRUE by default == assumed!

Only other bits:

1. Why is there a default for the reformat argument in pv_query? It's not exposed, afaik?
2. The vignette and docs will need updating to reflect the tweaks in output format/support for multiple params.